Sources/Moya/TargetType.swift

```diff
@@ -30,37 +24,42 @@ public protocol TargetType {
 }
 
 public extension TargetType {
+    var defaultParameterEncoding: ParameterEncoding {
```

Comment:
I think we can remove the `defaultParameterEncoding` from our codebase - I meant that someone can implement it in their own codebase if its convenient for them.